stdlib/DocPackageInfo.pkl

```diff
@@ -143,7 +143,7 @@ issueTracker: Uri
 /// The packages depended on by this package.
 ///
 /// Used to display package dependencies and to create documentation links.
-/// Set automatically for Pkl Hub packages.
+/// Set automatically for packages that are part of a docsite.
```

Comment:
Is it "a site of doc" or "a site of docs"? (Genuinely don't know; this could be math (US) vs maths (UK) thing); doc**s**site?

Reply:
I think of this as "A site of docs", or maybe "A documentation site".